compoments/report-repair/summary.js

```diff
@@ -14,20 +15,34 @@ const Summary = ({values, submit, goToStep}) => {
   const repairDetailsSummary = [
     { pageName:'Where is the problem?', value: values.repairLocation?.display, link:'repair-location'},
     { pageName:'What is the problem?', value: values.repairProblem?.display, link: 'repair-problems'},
-    { pageName:'What best describes the problem?', value: values['repairProblemBestDescription']?.display, link: 'repair-problem-best-description'},
     { pageName:'Description', value: values.description?.text, link:'repair-description'}
   ]
-  if (!isMvpReleaseVersion()) {
+  if (!isCommunal) {
```

Comment:
Both 'Repair Image' and 'What best describes the problem?' (line [27](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/81/files#diff-f11727ef3357684463ff5601dabb3151b2875be0272404536ef72a3c10373a11R27), moved from original location on line [17](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/81/files#diff-f11727ef3357684463ff5601dabb3151b2875be0272404536ef72a3c10373a11L17)) should be included in the communal (and non-communal) summary pages.
Neither should be restricted to non-communal summaries.
Please re-instate their displays on all summaries (or let me know if I've misunderstood something).

![image](https://user-images.githubusercontent.com/1875867/199964185-57b060ec-c40c-4fcc-987f-90ad197b3b6a.png)

Reply:
Thanks. I'd gone from the image in the ticket which looks out of date. Will follow the prototype.

Reply:
The value 'Communal area' for 'Where is the problem?' - this is picked up during the normal flow in the prototype - it is picked up as an option. Will this always be an option or should we be hard coding this in the summary page for communal repairs?

Reply:
This is based on the `repairTriage` API request, which is intercepted within the Cypress tests and will be the value(s) provided by the interception (i.e. [these](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/blob/main/tests/fixtures/repairTriageOptions.json))

Reply:
That's good, that 's where it's currently coming from.